cloud-sql-proxy.config.example.toml

```diff
@@ -0,0 +1,42 @@
+# Comment are supported in toml
```

Comment:
Let's remove this file and add a few more examples to the help command. Also, some kind of entry in the README (however short) is in order.

Reply:
removed! Added small info to README and an example in the help command which tells the user that they can also set other flags in the config file